ferrischat_webserver/src/users/get_user.rs

```diff
@@ -65,8 +65,8 @@ pub async fn get_user(req: HttpRequest, auth: crate::Authorization) -> impl Resp
                 } else {
                     None
                 },
-                flags: user.flags,
-                discriminator: 0
+                discriminator: user.discriminator,
+                flags: 0,
```

Comment:
Why set flags to 0 here instead of passing through from the DB user object returned?

Reply:
whoops, must have overlooked that on merge